src/app/environments/environments.ts

```diff
@@ -4,12 +4,12 @@
 
 export class environment {
     public static production = false;
-    public static baseUrl = 'https://cert-gen-jboss.herokuapp.com/';
+    public static baseUrl = 'https://jbossnewserver.herokuapp.com';
```

Comment:
Is this your fork's branch deployed?

Reply:
No, actually It's **tcmals** fork. 
His pull request is not merged yet, so I deployed it on my own heroku to use ```api/org/certificates``` route in this pr. [visiting-card-server](https://github.com/JBossOutreach/certificate-generator-server/)

Reply:
Changing it back to ```https://cert-gen-jboss.herokuapp.com```

Reply:
Yeah, Just checking on that.
We will merge that one soon.
Nice work. :wink: